linker.ld

```diff
@@ -20,4 +20,4 @@ SECTIONS
 
    /DISCARD/ : { *(.comment) *(.gnu*) *(.note*) *(.eh_frame*) }
 }
-__bss_size = (__bss_end - __bss_start)>>3;
+__bss_size = (__bss_end - __bss_start)>>3; /* size of bss in BYTES */
```

Comment:
wait if this is in bytes, that makes it sounds like bss_end and bss_start are in bits?